src/software/new_geom/util/distance.cpp

```diff
@@ -51,7 +53,7 @@ double distance(const Segment &first, const Point &second)
 
 double distance(const Point &first, const Polygon &second)
 {
-    if (second.contains(first))
+    if (contains(second, first))
```

Comment:
out of scope but for the "real" distance functions we do (first, second) as follows: (line, point), (point, segment), (point, polygon), (point circle) which is inconsistent. To me, it makes more sense to put point second in each case because it is always the "smaller" geom type. This would also make this line become contains(first, second) which is much more pleasant. thoughts? @MathewMacDougall @jonathanlew @akhilveeraghanta

Reply:
`distance` (and all functions for which argument order shouldn't matter) implement the function both ways, i.e. `distance(Polygon,Point)` and `distance(Point,Polygon)` are both valid.

Reply:
yeah thats a good point, I prefer that as well, its sort of a psuedo-convention that doesn't need to be strictly enforced, but would be nice to have an expected ordering of the arguments